pkg/k8s/api_test.go

```diff
@@ -203,7 +203,7 @@ status:
 		},
 		{
 			desc:     "Pod init container terminated with signal",
-			expected: "Init:Signal:9",
+			expected: "Running",
```

Comment:
This on the other hand seems congruent with the fact that the "signal" field is no longer taken into account :+1: